pkgs/tools/misc/wacomtablet-kcm/default.nix

```diff
@@ -0,0 +1,32 @@
+{
+  stdenv, mkDerivation, fetchurl, extra-cmake-modules, qtx11extras,
```

Comment:
You don't actually use stdenv, except for meta. Perhaps expect lib here and use that directly for meta